spec/views/trainees/personal_details/show.html.erb_spec.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+describe "trainees/personal_details/show.html.erb" do
+  context "with a trainee with no degree" do
+    let(:trainee) { create(:trainee) }
+
+    before do
```

Comment:
You have the same `before` block in your second context too so you can actually declare it just once outside of them both.

Reply:
thanks, fixed!